IssueTracker/IssueTracker.UnitTest/BaseClass.cs

```diff
@@ -8,10 +8,13 @@ public abstract class BaseClass
 {
     public static IIssueRepository IssueData = null!;
     public static TestContext TestContext { get; set; } = null!;
+    internal static string IdFileTest { get; set; } = String.Empty;
+    internal static Stream FileStreamTest { get; set; } = null!;
     public static DataTable? TestDataTable { get; set; }
     public static IParticipantRepository ParticipantData = null!;
```

Comment:
Consistency!!!! Use the same naming format everywhere.

Reply:
These elements were removed. I created an Object of the type File.

Reply:
The comments are always about the last line from the code snippet.

Reply:
i changed in IssueRepository and ParticipantRepository